build_and_release.sh

```diff
@@ -72,5 +72,5 @@ if gh release view "$GH_RELEASE_TAG" >/dev/null; then
   gh release upload "$GH_RELEASE_TAG" --clobber -- "${assets[@]}"
 else
   echo "creating release and uploading assets..."
-  gh release create "$GH_RELEASE_TAG" $prerelease $draft_release --title="${GITHUB_REPOSITORY#*/} ${GH_RELEASE_TAG#v}" --generate-notes -- "${assets[@]}"
+  gh release create "$GH_RELEASE_TAG" $prerelease $draft_release --title="${GH_RELEASE_TITLE} ${GH_RELEASE_TAG#v}" --generate-notes -- "${assets[@]}"
```

Comment:
**Would someone expect full control over the release title if `GH_RELEASE_TITLE` is set or not?**

This is the big question on my mind as `build_and_release.sh` doesn't use the `GH_RELEASE_TITLE` as-is but treats this more like a prefix.

If these changes had it such that it was full control over the title, then it means users would need a way to set this such that `INPUT_TAG` could be used after it has been evaluated above.  That's when this starts getting complicated.

Otherwise, should we rename this variable to avoid confusion?

@manue1 : what do you think?

Reply:
You're right, I actually missed on the first glance that the title string also included the tag. I have renamed it to title prefix in [38be18e](https://github.com/cli/gh-extension-precompile/pull/45/commits/38be18eca3ec6d4809b2ffa0ad2c678c72b92a12) for clarity. Let me know what you think.